src/content/self-hosted/digitalocean.mdx

```diff
@@ -29,8 +29,7 @@ In the rest of this guide, we will refer to this subdomain as `DOMAIN`.
 
 ### Deploy a Kubernetes cluster
 
-If you are not familiar with this step, we recommend that you follow DigitalOcean's [cluster creation guide](https://www.digitalocean.com/docs/kubernetes/how-to/create-clusters/).
-Okteto supports Kubernetes versions 1.22 through 1.24.
```

Comment:
I don't think we should remove this. We want this guide to be a single doc the installer can follow, not to have to jump back and forth.

Reply:
I think we should also add a recommendation on which type of instance we recommend for every cloud provider, using the specific cloud nomenclature.